examples/step-33/step-33.cc

```diff
@@ -201,14 +201,14 @@ namespace Step33
     }
 
```

Comment:
There is one thing one could address with this code: deal.II vector now also provide an operator[], so that code could be simplified. Do you want to try? (I will merge this PR anyway because it is unrelated to the proposed patch.)